rskj-core/src/main/java/co/rsk/rpc/modules/trace/TraceModuleImpl.java

```diff
@@ -157,6 +157,24 @@ public JsonNode traceFilter(TraceFilterRequest traceFilterRequest) throws Except
         return OBJECT_MAPPER.valueToTree(traces);
     }
 
+    @Override
+    public JsonNode traceGet(String transactionHash, List<String> positions) throws Exception {
+        TraceGetRequest request = new TraceGetRequest(transactionHash, positions);
+
+        TransactionInfo txInfo = this.receiptStore.getInMainChain(request.getTransactionHashAsByteArray(), this.blockStore).orElse(null);
+
+        if (txInfo == null) {
+            logger.trace("No transaction info for {}", transactionHash);
+            return null;
+        }
+
+        Block block = this.blockchain.getBlockByHash(txInfo.getBlockHash());
+
+        List<TransactionTrace> traces = buildBlockTraces(block);
+
+        return OBJECT_MAPPER.valueToTree(traces.get(request.getTracePositionsAsListOfIntegers().get(0)));
```

Comment:
can we properly validate input params that are being provided, meaning can we throw an invalid param error, if, for instance, `transactionHash` is `null` / empty string or `getTracePositionsAsListOfIntegers()` returns an empty list? Actually should we return an error if the list has more than one index? I assume right now it'd simply crash with the internal server error?